layouts/shortcodes/dbm-sqlserver-agent-config-examples.md

```diff
@@ -137,3 +137,16 @@ instances:
     password: '<PASSWORD>'
     reported_hostname: products-replica-1
 ```
+
+### Using port autodiscovery
```

Comment:
```suggestion
### Discovering ports automatically
```

changing this to disambiguate from container autodiscovery

Reply:
@cswatt this is verbiage specific to SQL Server, why change it?